### PR TITLE
WIP: mirage-conduit: better constraints

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: packages/mirage-conduit/mirage-conduit.3.0.0/opam [hunk 1]
```diff
@@ -15,19 +15,22 @@ build: [
...
+  "ppx_sexp_conv" {< "v0.14.2"}
+  "cstruct" {>= "3.0.0" & < "4.0.0"}
   "mirage-types-lwt" {>= "3.0.0" & < "3.7.0"}
   "mirage-flow-lwt" {>= "1.2.0"}
+  "dns"
+  "dns-lwt"
   "mirage-dns" {>= "3.0.0" & < "4.0.0"}
-  "conduit-lwt" {< "3.0.0"}
+  "conduit" {= "1.0.0"}
+  "conduit-lwt" {= "1.0.0"}
```
Comment (kit-ty-kate) [inline packages/mirage-conduit/mirage-conduit.3.0.0/opam:26]: Is that necessary? Would it be only available with `1.0.0` before? I'd rather avoid strict constraints in opam-repo

Comment (samoht) [reply packages/mirage-conduit/mirage-conduit.3.0.0/opam:26]: All the versions of `mirage-conduit` have been released in sync with `conduit` (but with a different version number). I think it's better to keep them constrained, especially if that help opam 2.0's solver.

**Comment by samoht** [comment]
Also: I'm still trying to understand what constraint I could simplify to make opam 2.0's solver happier. so I'm turning that PR to draft mode.

**Comment by avsm** [comment]
I'll close this for now @samoht -- feel free to reopen when you want to experiment with constraints again.

**Pull request closed without merging**